### PR TITLE
renamed components

### PR DESCRIPTION
**Pull request by philamwild**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -3,18 +3,21 @@ import TopBar from "./components/TopBar";
 import MainPage from "./components/MainPage";
 import Footer from "./components/Footer";
 import { Switch, Route } from "react-router-dom";
-import MoviePageFilterByTitle from "./components/MoviePageFilterByTitle";
+import MovieSheet from "./components/MovieSheet";
 import "./App.css";
-import ListMovies from "./components/ListMovies";
+import ListMoviesByGenre from "./components/ListMoviesByGenre";
```
Comment (LoicBrassart) [inline frontend/src/App.js]: Shouldn't it be something like `Movies` ?

File: frontend/src/App.js [hunk 2]
```diff
@@ -3,18 +3,21 @@ import TopBar from "./components/TopBar";
...
   return (
     <div className="App">
       <TopBar />
       <Switch>
         <Route exact path="/" component={MainPage} />
-        <Route path="/movieSheet" component={MoviePageFilterByTitle} />
-        <Route path="/listMovies/:genreName" component={ListMovies} />
+        <Route path="/movieSheet" component={MovieSheet} />
+        <Route
+          path="/listMoviesByGenre/:genreName"
```
Comment (LoicBrassart) [inline frontend/src/App.js]: `/genre/:genreName` would be a shorter/better path for this route, don't you think ?

File: frontend/src/App.js (hunk 1 above)
Comment (philamwild) [reply frontend/src/App.js]: Hello, for us just "Movies" would bring more confusion than understanding.... it a real list of movies in range by genre. For us it's quite clear

File: frontend/src/App.js (hunk 2 above)
Comment (philamwild) [reply frontend/src/App.js]: .... don't really know what to say. We think the code is easier to understand if path has same same as component isn't?

**Comment by LoicBrassart** [comment]
Didn't see @philamwild's answer, sorry
My last comments were suggestions, if that's ok for you it's fine by me
Solve your conflicts, and here we go ! \o/

**Pull request closed without merging**